Header header
sensor_msgs/LaserScan scan
float32[] image_point_x
float32[] image_point_y
float32[] distance
